packages/integration-tests/helpers/rootChainHelper.js

```diff
@@ -129,13 +130,13 @@ async function approveERC20 (
 }
 
 async function depositEth (rootChain, childChain, address, amount, privateKey) {
-  const depositTx = transaction.encodeDeposit(address, amount, transaction.ETH_CURRENCY)
+  const depositTx = transaction.encodeDepositAld(address, amount, transaction.ETH_CURRENCY)
   return rootChain.depositEth(depositTx, amount, { from: address, privateKey })
 }
 
 async function depositToken (rootChain, childChain, address, amount, currency, privateKey) {
-  const depositTx = transaction.encodeDeposit(address, amount, currency)
-  return rootChain.depositToken(depositTx, { from: address, privateKey })
+  const depositTx = transaction.encodeDepositAld(address, amount, currency)
+  return rootChain.depositToken(`0x${depositTx.toString('hex')}`, { from: address, privateKey })
```

Comment:
Can you try reverting this change? It should work now, and it will be consistent with `depositEth()` above